cue/schemas/panels/table/table.cue

```diff
@@ -0,0 +1,33 @@
+// Copyright 2024 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import "strings"
+
+kind: "Table"
+spec: close({
+	density?: "compact" | "standard"
+	columns?: [...#columns]
+})
+
+#columns: {
+	name:               strings.MinRunes(1)
+	header?:            string
+	headerDescription?: string
+	cellDescription?:   string
+	align?:             "left" | "center" | "right"
+	enableSorting?:     bool
+	width?:             number | "auto"
+	hide?:              bool
+}
```

Comment:
(Putting this unrelated comment here just to ease the discussion, as github doesn't allow to create threads from standalone comments...)

As discussed offline, it's too bad that in the case of this panel (& others actually) we have some fields of the query that are useless but still displayed: 
![2024-07-24_10h25_50](https://github.com/user-attachments/assets/02d8945b-062c-4514-980a-e630e1064b0f)

- **Legend name**: ideally it should either serve a purpose (e.g as a default name for the "value" column generated for this query) or hide it if not applicable (but may be not so simple to implement + doesnt solve the problem to have a "zombie" attribute in the model)
- **Min step**: This I guess the proper solution is out of scope of this panel solely, we already discussed the fact to split the current Prometheus timeseries query plugin in 2 different query plugins, one for instant queries & one for range queries, this way the instant query one could just not have this attribute in its model.